plugins/by-name/venv-selector/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  config,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "venv-selector";
+  package = "venv-selector-nvim";
+
+  maintainers = [ lib.maintainers.GaetanLepage ];
+
+  dependencies = [
+    "fd"
+  ];
+
+  settingsExample = {
+    name = [
+      "venv"
+      ".venv"
+    ];
+    dap_enabled = true;
+    pyenv_path = lib.nixvim.nestedLiteralLua "vim.fn.expand('$HOME/.pyenv/versions')";
+  };
+
+  extraConfig = cfg: {
+    warnings = lib.nixvim.mkWarnings "plugins.venv-selector" (
+      lib.map
+        (pickerName: {
+          when = (cfg.settings.picker or null) == pickerName && !config.plugins.${pickerName}.enable;
+          message = ''
+            You have to enable `plugins.${pickerName}` as `settings.picker` is set to `"${pickerName}"`.
+          '';
+        })
+        [
+          "telescope"
+          "fzf-lua"
+        ]
+    );
+  };
```

Comment:
Kinda like this pattern, feel like we could do something for more plugins like that.

Reply:
Yes, maybe introducing some helper for instance. I have no clear idea yet though.